components/automate-ui/src/app/modules/infra-proxy/clients/clients.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { combineLatest } from 'rxjs';
+import { NgrxStateAtom } from 'app/ngrx.reducers';
+import { LayoutFacadeService, Sidebar } from 'app/entities/layout/layout.facade';
+import { filter } from 'rxjs/operators';
+import { isNil } from 'lodash/fp';
+import { EntityStatus } from 'app/entities/entities';
+import { GetClients } from 'app/entities/clients/client.action';
+import { Client } from 'app/entities/clients/client.model';
+import {
+  allClients,
+  getAllStatus as getAllClientsForOrgStatus
+} from 'app/entities/clients/client.selectors';
+
+
+@Component({
+  selector: 'app-clients',
+  templateUrl: './clients.component.html',
+  styleUrls: ['./clients.component.scss']
+})
+
+export class ClientsComponent implements OnInit {
+  @Input() serverId: string;
+  @Input() orgId: string;
+
+  public clients: Client[] = [];
+  public clientsListLoading = true;
+
+  constructor(
+    private store: Store<NgrxStateAtom>,
+    private layoutFacade: LayoutFacadeService
+  ) { }
+
+  ngOnInit() {
+    this.layoutFacade.showSidebar(Sidebar.Infrastructure);
+
+    this.store.dispatch(new GetClients({
+      server_id: this.serverId, org_id: this.orgId
+    }));
+
+    combineLatest([
+      this.store.select(getAllClientsForOrgStatus),
+      this.store.select(allClients)
+    ]).pipe(
+      filter(([getClientsSt, _allClientsState]) =>
+      getClientsSt === EntityStatus.loadingSuccess && !isNil(_allClientsState))
+    ).subscribe(([ _getClientsSt, allClientsState]) => {
```

Comment:
Will need someone else to double check me on this, but from what i understand, since we are subscribing, we need to make sure we're unsubscribing in some way prior to component unmount.

@msorens - is this a candidate for `takeUntil(isDestroyed)` ?

Reply:
@SEAjamieD I will add this, actually, I thought we are going to add the `create` and `delete` code part in this file soon so that time we can add `takeUntil(isDestroyed)`.

Reply:
@SEAjamieD Thanks for raising that--the answer is a resounding "Yes!". 
@vinay033 In the future, let's please ensure that when you add a subscription you also add the proper lifecycle management (`takeUntil(isDestroyed)`).